src/models/park.py

```diff
@@ -1,14 +1,15 @@
-from sqlalchemy import Column, Integer, String, LargeBinary, ForeignKey, Identity
+from sqlalchemy import Column, Integer, String, Float, ForeignKey, Identity, LargeBinary
 from ..db.database import Base
 
+
 class Park(Base):
 
     __tablename__ = "park"
 
-    id = Column("id", Integer, Identity(start=1, increment=1), primary_key=True)
-    name = Column("name", String)
-    picture = Column("picture", LargeBinary)
-    location = Column("location", String)
-    total_spots = Column("total_spots", Integer)
-    owner = Column(Integer, ForeignKey("client.id"))
-
+    id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
+    name = Column(String, nullable=False)
+    picture = Column(LargeBinary, nullable=True)
+    latitude = Column(Float, nullable=True)
+    longitude = Column(Float, nullable=True)
+    rtsp_url = Column(String, nullable=True)
```

Comment:
The spots where suppose to be here no?

Reply:
Yeah I think so, it should exist there a list of parking spots, and in the database diagram that @WildBunnie made there isn't an entity for the parking spot.